packages/functional-tests/tests/signin/signIn.spec.ts

```diff
@@ -33,7 +34,7 @@ test.describe('severity-2 #smoke', () => {
       await login.clickForgotPassword();
 
       //Verify reset password header
-      await resetPassword.resetPasswordHeader();
+      expect(await resetPassword.resetPasswordHeader()).toBe(true);
```

Comment:
Should this use `toBeVisible()`?

Reply:
This method doesn't return a locator unfortunately, so toBeVisible() isn't available. It also has some React vs Backbone logic that would need to be worked through and propagated. kindof a large scope for this PR.